### PR TITLE
add ascii to functional

### PR DESCRIPTION
**Pull request by heavyk**
ran into this when trying to install dlangide

fixes https://github.com/buggins/dlangide/issues/398

### CODE REVIEW

File: std/functional.d [hunk 1]
```diff
@@ -68,6 +68,7 @@ module std.functional;
 
 import std.meta : AliasSeq, Reverse;
 import std.traits : isCallable, Parameters;
+import std.ascii;
```
Comment (atilaneves) [inline std/functional.d:71]: Surely this can't be the way forward. Why exactly can't this be local?

Comment (thewilsonator) [reply std/functional.d:71]: See the other two comments.

Comment (atilaneves) [reply std/functional.d:71]: I did. Still not 100% sure.

**Comment by heavyk** [comment]
ok, just let me know what you want me to change when it's decided :)

**Comment by heavyk** [comment]
@NilsLankila yes that seems quite appropriate, actually. nevermind then :)

**Pull request closed without merging**